data/2023/2023-04-25/post_vars.json

```diff
@@ -0,0 +1,5 @@
+{
+  "article_link": "https://nrennie.rbind.io/blog/web-scraping-rvest-london-marathon/",
+  "pic1_alt": "Hex logo of LondonMarathon R package. Background of the hex is an ariel photo of the Thames River and downtown London, and LondonMarathon is written on it, overlaid in red font.",
```

Comment:
```suggestion
  "pic1_alt": "Hex logo of LondonMarathon R package. Background of the hex is an aerial photo of the Thames River and downtown London, and LondonMarathon is written on it, overlaid in red font.",
```

Reply:
Thank you! Indeed I was not referring to the mermaid.